distribution/changelog.txt

```diff
@@ -20,6 +20,7 @@
 - Fix: [#21171] [Plugin] Crash creating entities with no more entity slots available.
 - Fix: [#21178] Inca Lost City’s scenario description incorrectly states there are height restrictions.
 - Fix: [#21198] [Plugin] Setting brake or booster speeds on a tile element doesn’t work.
+- Fix: [#21291] Hungry guests heading to any flat ride do not count for warning threshold.
```

Comment:
```suggestion
- Fix: [#21291] Hungry guests heading to any flat ride do not count for warning threshold (original bug).
```
Since this is in the original game, we typically annotate that.